src/components/DataTable/index.tsx

```diff
@@ -11,6 +11,7 @@ export type DataTableProps<Data extends object> = {
   data: readonly Data[]
   columns: readonly Column<Data>[]
   renderRowSubComponent?: SubComponent<Data>
+  onClickRow?: (row: Data) => void
```

Comment:
a cell is `Data`, not a row. does this typecheck?

Reply:
it does since `data: Data[]`